ophasebase/models.py

```diff
@@ -75,6 +75,7 @@ class Meta:
     start_date = models.DateField(verbose_name=_('Beginn'))
     end_date = models.DateField(verbose_name=_('Ende'))
     is_active = models.BooleanField(default=False, verbose_name=_('Aktiv?'))
+    contact_email_address = models.CharField(max_length=100, verbose_name=_('Kontaktadresse Leitung'))
```

Comment:
There is an [EmailField] (https://docs.djangoproject.com/en/1.10/ref/models/fields/#emailfield) in Django for email addresses.